website/docs/developer-docs/docs/writing-documentation.md

```diff
@@ -54,8 +54,6 @@ In addition to following the [Style Guide](./style-guide.mdx) please review the
 
 For new integration documentation, please use the Integrations template in our [Github repo](https://github.com/goauthentik/authentik) at `/website/integrations/template/service.md`.
 
-- Make sure to add the service to a fitting category in `/website/sidebarsIntegrations.js`. If this is not done the service will not appear in the table of contents to the left.
```

Comment:
Why remove this? We get a lot of PRs for Integration Guides in which the contributor forgot to add the Guide to the sidebar file. I see we need to rename the file, but let's not remove whole thing. Also, thanks for catching this update!